### ISSUE TITLE
Add assistive text / title to new status bar controls

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#2215]
__Is your feature request related to a problem? Please describe.__

We use assistive text (title text) for most of our controls in the app. However, the new buttons do not ship with these texts, leaving users in the dark what they do:

![capture ce0lUd_optimized](https://user-images.githubusercontent.com/58601/114840434-19b8c100-9dd7-11eb-987d-b432aab3b1d0.gif)

_See how the log button actually has an assistive text._

__Describe the solution you'd like__

Add assistive text to all status bar buttons. Always add assistive title text to new buttons created, unless they are super clear ("O.K."). 

* `XML`: "Toggle XML view" or the like
* `Camunda Engine`: _?_

__Describe alternatives you've considered__

No alternative.

__Additional context__

Related to our ongoing accessibility efforts / discussion.
